### PR TITLE
PR 2

### PR DESCRIPTION
**Pull request by IndreshKry**
Able to merge.
Did updation using git pull --rebase

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -248,6 +248,7 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
                      <div class="col-md-12">
                         <div class="row">
                            <div class="portfolio-items isotopeWrapper clearfix" id="4">
+
```
Comment (abhinavjain241) [inline index.html]: There are extra spaces that have been added everywhere. While getting the diff, git also considers these spaces to be added as a change, so remove these spaces.

File: index.html [hunk 2]
```diff
@@ -337,6 +356,17 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
                                        <span>Member, Embedded Team</span>
                                     </div>
                                  </div>
+                              </article>
+
```
Comment (abhinavjain241) [inline index.html]: Did you add Shivang's information also along with your commit? He already did that.

Comment (IndreshKry) [inline index.html]: Can I edit it from here or do I have to send a new PR??

Comment (abhinavjain241) [inline index.html]: You can edit it from here. No need to send a new PR. Also, if you keep doing commits to your local repo, they will automatically be added to the PR.

**Comment by abhinavjain241** [comment]
@IndreshKry : Still has conflicts. Please rebase and commit.

**Comment by pranitbauva1997** [comment]
Closing this PR due to inactivity. If you want to work on it, feel free to re-open it!

**Pull request closed without merging**